.github/workflows/channels-ci.yml

```diff
@@ -113,7 +113,7 @@ jobs:
         working-directory: webapp/channels
         run: |
           npm run check-types
-  tests:
+  test:
     runs-on: ubuntu-22.04
```

Comment:
We can maybe use ubuntu-8-cores to run the tests even faster.

Reply:
I was assuming this could take a long time but now checked and it seems like it takes 4m-7m so, I think default runner would be okay.